### PR TITLE
CMR-9102 Upgrades to support Java17

### PR DESCRIPTION
**Pull request by jaybarra**
Updates libraries and associated calls to work with Java17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,44 +32,32 @@ Search API provides access to this metadata.
...
+- `gcc` and `libc`
 - Docker
 
 ## Obtaining the Code
 
 You can get the CMR source code by cloning the repository from GitHub:
 
-```
-$ git clone git@github.com:nasa/Common-Metadata-Repository.git cmr
+``` sh
```
Comment (jceaser) [inline README.md:49]: why sh?

File: common-app-lib/project.clj [hunk 2]
```diff
@@ -16,15 +16,14 @@
                  [nasa-cmr/cmr-redis-utils-lib "0.1.0-SNAPSHOT"]
                  [nasa-cmr/cmr-transmit-lib "0.1.0-SNAPSHOT"]
                  [org.clojure/clojure "1.10.0"]
-                 [org.pegdown/pegdown "1.6.0"]
-                 [ring/ring-core "1.9.6"]
+                 [com.vladsch.flexmark/flexmark-all "0.64.0"]
+                 [ring/ring-core "1.9.2"]
```
Comment (jceaser) [inline common-app-lib/project.clj:20]: we are going down not up?

File: README.md (hunk 1 above)
Comment (jaybarra) [reply README.md:49]: That's only for markdown display purposes, it's not part of the command.

File: common-app-lib/project.clj (hunk 2 above)
Comment (jaybarra) [reply common-app-lib/project.clj:20]: Good catch, I'll bump that to match

**Review by daniel-zamora — APPROVED** [review]

File: system-int-test/src/cmr/system_int_test/utils/queue.clj [hunk 3]
```diff
@@ -47,4 +47,4 @@
   (queue-message-count "")
 
 
-  )
\ No newline at end of file
+  )
```
Comment (jceaser) [inline system-int-test/src/cmr/system_int_test/utils/queue.clj:50]: should this be on line 47?

**Review by jceaser — APPROVED** [review]

**Pull request merged**